util/components.js

```diff
@@ -0,0 +1,19 @@
+import { By } from "selenium-webdriver";
+
+export async function setOEBInputValueById(driver, id, value, fieldType='input') {
+    const linkDescOebInput = await driver.findElement(By.id(id));
+    const linkDescField = await linkDescOebInput.findElement(
+      By.tagName(fieldType)
+    );
+    await linkDescField.clear()
+    await linkDescField.sendKeys(value);
+
+}
+
+export async function setOEBInputValueByCSS(driver, cssSelector, value) {
+  const linkDescOebInput = await driver.findElement(
+    By.css(`oeb-input[label=${cssSelector}]`)
+  );
+  const linkDescField = await linkDescOebInput.findElement(By.tagName("input"));
```

Comment:
Why no `clear` here? Generally it seems this does the same as the other method, just with another selector. Maybe a third method that extracts the duplicate would make sense?

Reply:
I added it here as well